Makefile

```diff
@@ -14,8 +14,8 @@
 
 .PHONY: all rbdplugin
 
-IMAGE_NAME=csi_images/rbdplugin
-IMAGE_VERSION=latest
+IMAGE_NAME=quay.io/k8scsi/rbdplugin
```

Comment:
we cannot push to that repo

Reply:
All CSI images are stored at quay, but I agree if we need to store rbdplugin where we can push it.

Reply:
created quay.io/cephcsi, let's use it